compiler/compile/OMRCompilation.cpp

```diff
@@ -441,10 +441,9 @@ OMR::Compilation::Compilation(
    // NextGenHCR is disabled, as they are enabled for it
```

Comment:
Does the comment one line above this (L440) need to change to:   "Do not default OSR on if:"  ?